### PR TITLE
Add ppc64le and osx

### PR DESCRIPTION
**Pull request by Eigenstate**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
This is not how you enable ppc64le...

**Comment by Eigenstate** [comment]
What's the best way? I'm using the [conda-forge documentation ](https://conda-forge.org/docs/maintainer/conda_forge_yml.html) for clues but I know it can be sometimes out of date

File: conda-forge.yml [hunk 1]
```diff
@@ -5,3 +5,10 @@ max_py_ver: '37'
 max_r_ver: '35'
 travis:
   secure: {BINSTAR_TOKEN: HjhDPhowaeeFyzHaQ09iOBM6e9H7IK90ucA4tD0w2WmG4iW4I2a7ReGwgbnaHwOJ+C1oZS+zHGK2xmM3SBWDvCQi6xhVNxvKTE8kwr+WvguHBHf7ckv9Kb9rs5c4y2GNwN29S6ZpiX4xM+aHCLGQWMi4PXKf/yOHuUjbPWbZGQj8UjJnUvGYpA14qeKv8SaCD22xB7oQyL9WEAkMp/34kwfxyrlFgyBqkkNT9XFgM4rGhLCYJ38FRwCDZ1m/Im8ZBRjdEmyvftjozTSV5W98uaaVJOHum7B1fJen7XzWIaGs4GxBeWNaCXRsmvbwcFZ7wm5Gq0iFGFGxuguAskM3QJ0seVd3Nj7jwCPhJphxBo8YxhyyE52ubUxVsywbJJ20aHBpzXSwncroKEMicVs69LxrzeJqPIwJfpr7s+0uCAUuOYTnNBm15T9dG43NQIsZbPQm7R8F5F82bT1neZi82HTLSOlRt5o1CMVYqO5L9TJjArWR2RV68V/cH1aoq62apwfxxgXNhnzHR7qDYJSy9n6GS9P3PV0mX/olxUPc5jNLdC4ARPOJdnTBBT2vgWYPbRuO47Qc1sfwb/E1oTFjNf3znnr1r1OnB0USw9S2sbIoHFrgGuJS/eWmJTGfe6fRxTiLFpW+LERnrfrLpcJubN+S+ZsxuDYY7A8MD6U6beQ=}
+linux_ppc64le:
```
Comment (mariusvniekerk) [inline conda-forge.yml]: change this entire block to

```yaml
provider:
    linux_ppc64le: azure
```

Comment (mariusvniekerk) [reply conda-forge.yml]: and then rerender

Comment (Eigenstate) [reply conda-forge.yml]: Thanks!

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

**Comment by Eigenstate** [comment]
@conda-forge-admin please rerender

**Comment by mariusvniekerk** [comment]
@Eigenstate if you want the migrator to get to this one please add a line to https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators.xsh#L904-L932 in a PR.  That way the bot will issue pull requests for your dependencies

**Comment by Eigenstate** [comment]
Thanks! I'll do that once I get aarch64 to work.

**Pull request merged**